src/main/webapp/destinationCreation.js

```diff
@@ -22,31 +32,87 @@ function addScriptToHead() { // eslint-disable-line
 }
 
 function searchForPlace() { // eslint-disable-line
+  const element = document.querySelectorAll('#place');
```

Comment:
IDs are supposed to be unique, and since you want to have multiple divs, you should probably be using class to query for the elements, not ID